telemetry/prometheus/src/main/scala/io/buoyant/telemetry/prometheus/PrometheusTelemeter.scala

```diff
@@ -81,8 +81,16 @@ class PrometheusTelemeter(metrics: MetricsTree, private[prometheus] val handlerP
         (Seq("rt", "client"), labels0 :+ ("client" -> escapeLabelVal(id)))
       case Seq("rt", "client", "service", path) if !labelExists(labels0, "service") =>
         (Seq("rt", "client", "service"), labels0 :+ ("service" -> escapeLabelVal(path)))
+      case Seq("rt", "client", "failures", path) if !labelExists(labels0, "exception") =>
+        (Seq("rt", "client", "failures"), labels0 :+ ("exception" -> escapeLabelVal(path)))
+      case Seq("rt", "client", "exn", path) if !labelExists(labels0, "exception") =>
```

Comment:
What do you think about using the word `exceptions` instead of `exn`? To me, it seems better to have it spelled out so that its descriptive.

Reply:
I agree it would be more descriptive. The reason I kept it as `exn` is because that's the naming convention of the metric tree. I'm not sure if that would introduce any confusion when trying to debug and realizing that the node `exn` is in fact labeled as `exceptions` in the output.